control-plane/csi-driver/src/bin/controller/main.rs

```diff
@@ -111,12 +111,19 @@ async fn main() -> anyhow::Result<()> {
                 .help("Formatting style to be used while logging")
         )
         .arg(
-        Arg::new("ansi-colors")
-            .long("ansi-colors")
-            .default_value("true")
-            .value_parser(clap::value_parser!(bool))
-            .help("Enable ansi color for logs")
-    )
+            Arg::new("ansi-colors")
+                .long("ansi-colors")
+                .default_value("true")
+                .value_parser(clap::value_parser!(bool))
+                .help("Enable ansi color for logs")
+        )
+        .arg(
+            Arg::new("disable-stale-cleanup")
+                .long("disable-stale-cleanup")
+                .action(clap::ArgAction::SetFalse)
+                .value_name("BOOLEAN")
+                .help("Enable cleanup of the stale entries before controller publish")
```

Comment:
help is saying `Enable` while the arg name says disable.

Reply:
Removed this now as per above comments.